app/controllers/application_controller.rb

```diff
@@ -42,6 +42,10 @@ def render_json(data: {}, errors: [], status: :ok)
     }, status:
   end
 
+  def ensure_authenticated
+    render_json status: :unauthorized and return unless current_user
```

Comment:
Lets leave this for another PR (and avoid the `and` unless necessary)

Reply:
Sure,
Does the `render` method kills the request/response chain, if so then the `and return` is absurd unlike with the `redirect_to`.

Reply:
Render doesn't stop the request/response chain, to avoid a full `ìf...else....end` block while using the one liner syntax using the `and return` to kill the chain is recommended.
https://guides.rubyonrails.org/layouts_and_rendering.html#avoiding-double-render-errors

Reply:
```
def ensure_authenticated
  return unless current_user
  return render_json status: :unauthorized  
end
```